### ISSUE TITLE
New HA install, bad credentials error

### ISSUE THREAD

**Issue by danner26** [custom-components/hacs#136]
## Version of HACS
Latest version has been installed of both HA and HACS. 0.94.3 and 0.8.1 respectively. 

## Describe the bug
I have added the ~/.homeassistant/custom_components/
The configuration.yaml is in the same folder as the custom components ~/.homeassistant/
HACS is installed inside that custom_components folder

In my configuration.yaml, I have added: 
```
hacs:
  token: !secret hacs_token
```

Inside my secrets.yaml, I have added:
```
hacs_token: KEY_GENERATED_BY_GITHUB
```

When starting HA, the end result is always the following:
```
Traceback (most recent call last):
  File "/srv/homeassistant/bin/hass", line 10, in <module>
    sys.exit(main())
  File "/srv/homeassistant/lib64/python3.6/site-packages/homeassistant/__main__.py", line 396, in main
    exit_code = asyncio_run(setup_and_run_hass(config_dir, args))
  File "/srv/homeassistant/lib64/python3.6/site-packages/homeassistant/util/async_.py", line 29, in asyncio_run
    return loop.run_until_complete(main)
  File "/opt/rh/rh-python36/root/usr/lib64/python3.6/asyncio/base_events.py", line 454, in run_until_complete
    self.run_forever()
  File "/opt/rh/rh-python36/root/usr/lib64/python3.6/asyncio/base_events.py", line 421, in run_forever
    self._run_once()
  File "/opt/rh/rh-python36/root/usr/lib64/python3.6/asyncio/base_events.py", line 1426, in _run_once
    handle._run()
  File "/opt/rh/rh-python36/root/usr/lib64/python3.6/asyncio/events.py", line 127, in _run
    self._callback(*self._args)
  File "/home/ha/.homeassistant/custom_components/hacs/hacsbase.py", line 56, in startup_tasks
    repository = await self.aiogithub.get_repo("custom-components/hacs")
  File "/srv/homeassistant/lib64/python3.6/site-packages/backoff/_async.py", line 131, in retry
    ret = await target(*args, **kwargs)
  File "/home/ha/.homeassistant/custom_components/hacs/aiogithub.py", line 61, in get_repo
    raise AIOGitHubException(response["message"])
custom_components.hacs.aiogithub.AIOGitHubException: Bad credentials
```

I am not really sure why it says bad credentials.. Also, inside GitHub for verification: 
![image](https://user-images.githubusercontent.com/8165027/59776864-8ae1c880-9281-11e9-842a-736921033596.png)

## Debug log
HA wont start, so I cant get this information I don't believe. Correct me if I am wrong and I will give it a shot for you.

HASS log output:
https://pastebin.com/KNZXAvDY

**Comment by ludeeus** [comment]
That error say that GitHub does not accept your token, try generating a new one.

**Comment by danner26** [comment]
I've generated 3 tokens now, unfortunately no dice.

**Comment by ludeeus** [comment]
Try without using secrets, and putting quotes around the token

**Comment by danner26** [comment]
Great idea, so same Bad Credentials error. I tried the quotes in both the secret, and directly in the config file and no dice again.

So, I went ahead and I generated a new key again, put it directly into the configuration file with no quotes and it worked. So, I let HACS do what it needed to do and moved it back to the secret file. It worked perfectly.

No idea what the issue was, probably just me doing something stupid. Thank you!
